### DISCUSSION TITLE
How to use on replit

### DISCUSSION THREAD

**Discussion by anonymouse-cmd** [SwiftyProgrammer690/fireworks#5] [Q&A] [answered]
So, how would you use this on replit since it is different compared to desktop ides?

**Comment by SwiftyProgrammer690** [comment, marked as answer]
You would use their way of cloning, if you don't want to do their way, then you can go to a new repl and go to their shell/bash and enter the commands from there!

**Reply by anonymouse-cmd** [reply]
Thank you!
